src/actions/breakpoints/breakpointPositions.js

```diff
@@ -64,6 +75,10 @@ export function setBreakpointPositions(location: SourceLocation) {
 
     let positions = convertToList(results, sourceId);
     positions = await mapLocations(positions, getState(), source, sourceMaps);
-    return dispatch({ type: "ADD_BREAKPOINT_POSITIONS", positions, location });
+    dispatch({ type: "ADD_BREAKPOINT_POSITIONS", positions, location });
```

Comment:
Can we change this action to take just a `sourceId`? Reviewing this as-is, I was worried that this takes a whole location but we only cache on `sourceId`, so it could easily be a case where the cache key and the used data fall out of sync.